5.0/en/0x05-For-Users-Of-4.0.md

```diff
@@ -1,86 +1,84 @@
+While these guidelines remain an important reference, strict alignment introduced challenges, including less widely recognized terminology, duplication of similar requirements, and incomplete mappings. Version 5.0 moves away from this approach to improve clarity and relevance.
 
-### Moving away from Common Weakness Enumeration (CWE)
+### Moving Away from Common Weakness Enumeration (CWE)
 
-The [Common Weakness Enumeration (CWE)](https://cwe.mitre.org/) from MITRE is a useful way to map out different security weaknesses in software. There are some difficulties in using it including certain CWEs which are categories only and shouldn't be used for mapping, the difficulties in mapping certain existing requirements to a single CWE, and also the fact that there were some loose or inexact mappings in version 4.x of ASVS.
+The [Common Weakness Enumeration (CWE)](https://cwe.mitre.org/) provides a useful taxonomy of software security weaknesses. However, challenges such as category-only CWEs, difficulties in mapping requirements to a single CWE, and the presence of imprecise mappings in version 4.x have led to the decision to discontinue direct CWE mappings in version 5.0.
 
-## Rethinking the level definitions
+## Rethinking Level Definitions
 
-Version 4.x of the ASVS describes the levels as L1 - "Minimum", L2 - "Standard", and L3 - "Advanced" with the implication that all applications processing sensitive data should be at least L2.
+Version 4.x described the levels as L1 ("Minimum"), L2 ("Standard"), and L3 ("Advanced"), with the implication that all applications handling sensitive data should meet at least L2.
 
-There were various issues with this approach that version 5.0 was specifically designed to address.
+Version 5.0 addresses several issues with this approach:
 
-### Easier entry level
+### Improved Entry Level
 
-From feedback on the use (or non-use) of previous ASVS versions in industry, the single greatest problem that was identified was the double-edged sword of Level 1 having a large number of requirements (~120) but at the same time being considered the "minimum" level that is not good enough for most applications. This seemed to lead to organizations either giving up before they start or trying to implement a subset of the requirements without actually achieving Level 1, therefore reducing the sense of achievement and progress.
+Feedback indicated that the large number of Level 1 requirements (~120), combined with its designation as the "minimum" level, discouraged adoption. Organizations often either abandoned the standard or implemented only a subset of requirements, undermining the sense of progress and achievement.
```

Comment:
> Organizations often either abandoned the standard or implemented only a subset of requirements, undermining the sense of progress and achievement.

We should not serve this as a problematic fact. To take a subset of the requirements that are needed for the organization is encouraged in some other section.

Our goal is just improve the adoption and make the first step in easier.

Reply:
I made a change, let me know what yhou thin

Reply:
I think my comment is also valid for the new version.

We should not state any "facts" that it made it harder - there is no science or evidence to provide for that. We just have abstract feedback and the goal to make L1 focus clear and precise, and it resulted in ~70 requirements with a clear focus on the first layer of defense.

Reply:
Ok, tried again

Reply:
We can go with the last version
> Version 5.0 aims to lower this barrier to entry with less Level 1 requirements

Or it can be improved with the message
> Version 5.0 aims to lower this barrier by having mainly requirements for the first layer of defense for Level 1, resulting better better-focused and fewer Level 1 requirements.

Reply:
Tried again.